src/main/resources/alfio/templates/ticket.ms

```diff
@@ -86,7 +86,7 @@
 			</tr>
 			<tr>
 				<td>{{#i18n}}ticket.reference-number{{/i18n}}</td>
-				<td>{{ticket.uuid}}</td>
+				<td>{{ticket.publicUuid}}</td>
```

Comment:
we will need to handle the case where somebody has an overridden template definition